docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -0,0 +1,376 @@
+---
+id: proxy-and-tunnel
+title: Ory Proxy and Ory Tunnel
+---
+
+The CLI contains two important tools for local development and self-hosted instances
+of the Ory Identity Service (Kratos): 
+
+- Ory Proxy
+- Ory Tunnel
+
+Before you start using these tools, export the SDK of your Ory Cloud project. Run this command: 
+
+```shell
+export ORY_SDK_URL=https://{your-project-slug-here}.projects.oryapis.com
+
+# For a quick way to test, use the playground project: 
+export ORY_SDK_URL=https://playground.projects.oryapis.com
+```
+:::tip
+
+To get your project's SDK URL, sign in at
+[console.ory.sh](https://console.ory.sh/), select **Connect** from the left
+navigation panel, and copy the URL from the **SDK Configuration** section.
+
+:::
+
+## Ory Proxy 
+
+Ory Proxy is a reverse proxy deployed in front of your application. This allows the 
+Ory endpoints to be mirrored on the same domain as the app you're running. 
+Additionally, the Proxy converts sessions into JSON Web Tokens (JWTs) and ensures that cookies 
+and URLs are properly configured.
+
+:::info
```

Comment:
I vote for some diagrams to show how this works to make understanding easier because using ory proxy without additional context is quite hard. I can create this one for you to explain how it works.
Also, it would be great to add the motivation part why do I need to use ory proxy, for instance

- to prevent CORS issues
- to have integration with the local environment and ory cloud project

Also, I think that we need to have some examples of using ory proxy. Here're a couple of guides that can help you

- https://www.ory.sh/docs/guides/proxy
- https://www.ory.sh/docs/guides/proxy#ory-proxy-for-local-development

Reply:
Thanks, Andrew. If you want to whip up a diagram then I'm happy to include it :) 

When it comes to the motivation, have a look at lines 30-44 - I think this is covered. I can add a mention of the CORS aspect.

> Also, I think that we need to have some examples of using ory proxy.

This document gets removed and replaced by this one. Some of the content is moved from that document to the new one with some edits. 

When it comes to examples, the proxy is featured in almost all of the "protect page with login" examples and I don't really feel that this is a place to whip up yet another example - you'd have to describe a setup with an app, etc. This section is supposed to be all about CLI, no clutter ;)